src/messages/any/get.ts

```diff
@@ -0,0 +1,93 @@
+import axios from "axios";
+import { DEFAULT_API_V2 } from "../../global";
+import { getSocketPath, stripTrailingSlash } from "../../utils/url";
+import { BaseMessage, Chain, MessageType } from "../message";
+
+type MessageQueryResponse = {
+    messages: BaseMessage[];
+    pagination_page: number;
+    pagination_total: number;
+    pagination_per_page: number;
+    pagination_item: string;
+};
+
+type GetMessagesConfiguration = {
+    pagination?: number;
+    page?: number;
+    addresses?: string[];
+    channels?: string[];
+    chains?: Chain[];
+    refs?: string[];
+    tags?: string[];
+    contentTypes?: string[];
+    contentKeys?: string[];
+    hashes?: string[];
+    messageType?: MessageType;
+    startDate?: Date;
+    endDate?: Date;
+    APIServer?: string;
+};
+
+type GetMessagesParams = {
+    pagination?: number;
+    page?: number;
+    addresses?: string;
+    channels?: string;
+    chains?: string;
+    refs?: string;
+    tags?: string;
+    contentTypes?: string;
+    contentKeys?: string;
+    hashes?: string;
+    msgType?: string;
+    startDate?: number;
+    endDate?: number;
+};
+
+//TODO: Provide websocket binding (Refacto Get into GetQuerryBuilder)
```

Comment:
I think this is a separate implementation

Reply:
Yup it's for a futur implementation, but I removed this comment for now in [af8c768](https://github.com/aleph-im/aleph-sdk-ts/pull/117/commits/af8c768713282208300900b3f44b1dbd034a2f59)